packages/cli/src/commands/create/install-start.ts

```diff
@@ -0,0 +1,44 @@
+import * as prompts from '@clack/prompts';
+import { DEFAULT_VALUES, readFlag, type CreateOptions } from './options.js';
+import { assertNotCanceled } from '../../utils/tasks.js';
+
+export async function installAndStart(flags: CreateOptions) {
+  let installDeps = readFlag(flags, 'install');
+  let startProject = readFlag(flags, 'start');
+
+  if (installDeps === false) {
+    // the user doesn't want to install the dependencies, which means we can't start the project either
+    return { install: false, start: false };
+  }
+
+  if (startProject) {
+    // the user wants to start the project, so we also have to install the dependencies
+    return { install: true, start: true };
+  }
+
+  if (installDeps) {
+    if (startProject === false) {
+      // the user wants to install the dependencies but expicitly not start the project
+      return { install: true, start: false };
+    } else {
+      // the user explicitly wants to install the dependencies, but we don't know if they want to start the project
+      const answer = await prompts.confirm({
+        message: 'Start project?',
```

Comment:
If `--install` is provided, but not `--start`, we change the question to just `Start project?`.